### PR TITLE
Vet Tec: pre-filled bank information is not showing on the review page#16850

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
When completing a 0994 form, a user's bank information is filled out for them on the bank information page. Currently any pre-filled information on that page is not displayed upon reaching the review page.

Log in and complete a 0994 form as a user with pre-existing bank information.
Note the account information on the Bank Information page
Continue to the Review/submit page
Expand the Personal Information section
The same bank information should be displayed here, but none is shown.

## Testing done
- [x] local testing

## Screenshots
screen shot 2019-02-12 at 10 49 48 am

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by robfelder — APPROVED** [review]
LGTM! Just one question.

File: src/applications/edu-benefits/0994/components/PaymentReviewView.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  }
+
+  render() {
+    const { formData, name } = this.state;
+    let value = formData;
+    if (formData === undefined) {
+      const propName = `bank${name
+        .substring(0, 1)
+        .toUpperCase()}${name.substring(1)}`;
+      const prefillBankAccount = _.get(
```
Comment (robfelder) [inline src/applications/edu-benefits/0994/components/PaymentReviewView.jsx]: What happens if prefillBankAccount is empty?

**Review by robfelder — APPROVED** [review]

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/edu-benefits/0994/components/PaymentReviewView.jsx (hunk 1 above)
Comment (zurbergram) [reply src/applications/edu-benefits/0994/components/PaymentReviewView.jsx]: returns an empty object and the returns an empty string

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/edu-benefits/0994/utils.jsx [hunk 2]
```diff
@@ -13,3 +13,22 @@ export const srSubstitute = (srIgnored, substitutionText) => (
     <span className="sr-only">{substitutionText}</span>
   </span>
 );
+
+export const maskBankInformation = (string, unmaskedLength) => {
```
Comment (jbalboni) [inline src/applications/edu-benefits/0994/utils.jsx]: Might be useful to have unit tests for this

Comment (zurbergram) [reply src/applications/edu-benefits/0994/utils.jsx]: I have added an unit test

**Comment by zurbergram** [comment]
including fix for https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/16932

**Review by jbalboni — APPROVED** [review]

**Pull request merged**